ultravox/inference/ultravox_infer.py

```diff
@@ -55,7 +56,10 @@ def __init__(
         )
 
         processor = ultravox_processing.UltravoxProcessor(
-            audio_processor, tokenizer=tokenizer, stack_factor=model.config.stack_factor
+            audio_processor,
+            tokenizer=tokenizer,
+            stack_factor=model.config.stack_factor,
+            # TODO: how to pass audio_padding?
```

Comment:
Alright, so in the current version we decided not to modify `audio_padding` (it's always `"longest"`). So the question is: do we keep the machinery for it or not?

Con: more complex code.

Pro: we can come back to it at a later date if need be, plus, without it if we say we tried both, we can't point people to the code and say, look we tried it and you can try it too.

Reply:
Are you suggesting ripping it out instead? Can we still use other encoders if we do that?

Reply:
Let me rephrase.

I'm adding some functionality here that in the end doesn't do anything, unless we use `audio_padding=max_length`. I needed this to do the experiments, but now there's no point in having the options in the code base unless we 1) want to go back to it or 2) want to show others that we did it or that they can play with it.

So the question is do I keep this functionality or just assume `audio_padding` is always fixed to `"longest"`?

In either scenario I won't touch the existing encoders. They usually use `"longest"` by default as well.

Reply:
It's 5 LOC so I wouldn't sweat it too much either way. My suggestion would be to leave it in the processor to show our work but remove it from the configs (eg so they just always use the default).